src/components/DagComponent.js

```diff
@@ -489,14 +538,27 @@ class DagComponent extends React.Component {
     this.tooltip.style('left', `${event.pageX}px`).style('top', `${event.pageY + 10}px`);
   }
 
-  render() {
+  renderUi() {
+    return (
+      <div ref="dagWrapper" className="dagWrapper">
+        <svg id="graph" />
+        <div className="tooltip"></div>
+      </div>
+    );
+  }
+
+  renderNewUi() {
     return (
       <div ref="dagWrapper" className="dagWrapper">
         <svg id="graph" />
         <div className="tooltip"></div>
       </div>
     );
   }
```

Comment:
question: What changed here?

Reply:
At some point during development, we separated the returns, but we realized it wasn’t necessary. We’ve adjusted it now. Waiting for your review.